src/IHaskell/Types.hs

```diff
@@ -56,25 +65,32 @@ class IHaskellDisplay a where
 
 -- | Display as an interactive widget.
 class IHaskellDisplay a => IHaskellWidget a where
-  -- | Output target name for this widget. The actual input parameter should be ignored.
+  -- | Output target name for this widget. The actual input parameter should be ignored. By default
+  -- evaluate to "ipython.widget", which is used by IPython for its backbone widgets.
   targetName :: a -> String
+  targetName _ = "ipython.widget"
```

Comment:
Can we change this to use `Proxy` from `Data.Proxy` instead of ignoring the function parameter? (Maybe not, if it poses a 7.6.3 problem, but we should try.)